pkg/package-server/apiserver/scheme_test.go

```diff
@@ -0,0 +1,14 @@
+package apiserver
+
+//
+//
+//import (
+//"testing"
+//
+//"k8s.io/apimachinery/pkg/api/testing/roundtrip"
+//wardlefuzzer "k8s.io/sample-apiserver/pkg/apis/wardle/fuzzer"
```

Comment:
doubt we need the fuzzing since this is read only, but it might be worth looking at to see if it's useful. otherwise let's remove this.